src/amo/components/AddonMeta/styles.scss

```diff
@@ -60,6 +59,11 @@
     &:visited {
       color: $black;
     }
+
+    &:active,
```

Comment:
> I'm not sure why I have to repeat these here...

I think it's because [the whole anchor](https://github.com/mozilla/addons-frontend/blob/c3e8e3a73bca79e1d2f8dfbb14c80b4a41853d25/src/amo/components/AddonMeta/styles.scss#L57) is set to black and due to the specificity, the rule below it does not override it. I think you could remove the line I linked to and then you won't have to repeat [this rule](https://github.com/mozilla/addons-frontend/blob/c3e8e3a73bca79e1d2f8dfbb14c80b4a41853d25/src/amo/components/AddonMeta/styles.scss#L65).

An ampersand in Sass expands to the compiled selector after applying all nested brackets.

Reply:
Removing the rule for [`&`](https://github.com/mozilla/addons-frontend/blob/c3e8e3a73bca79e1d2f8dfbb14c80b4a41853d25/src/amo/components/AddonMeta/styles.scss#L57) doesn't work if I also remove the [repeated rule](https://github.com/mozilla/addons-frontend/blob/c3e8e3a73bca79e1d2f8dfbb14c80b4a41853d25/src/amo/components/AddonMeta/styles.scss#L65). The link remains black when hovered.

I also tried replacing the rule with the ampersands with

```
  a:link.AddonMeta-reviews-content-link,
  a:visited.AddonMeta-reviews-content-link {
    color: $black;
  }
```

to avoid overriding the whole anchor, but that also doesn't help and the link remains black when hovered over.

I cannot seem to get it to turn blue unless I do the repeated declaration as exists in the current patch. Does anyone have any other suggestions to try?

Reply:
I don't really see a way around it b/c you need override the specificity